src/models/dbModels.go

```diff
@@ -5,29 +5,18 @@ import (
 	"time"
 )
 
-type Watchlist struct {
-	ID            int    `gorm:"column:id" json:"id"`
-	UserID        int    `gorm:"column:user_id" json:"userID"`
-	WatchlistName string `gorm:"column:watchlist_name" json:"watchlistName"`
-}
-
-func (Watchlist) TableName() string {
-	return constants.WatchlistTable
-}
-
 type Users struct {
 	Id             uint16    `gorm:"primary_key;auto_increment" json:"id"`
-	UserName       string    `gorm:"column:username" json:"username"`
+	UserName       string    `gorm:"column:username" json:"userName"`
 	Name           string    `gorm:"column:name" json:"name"`
 	Email          string    `gorm:"column:email" json:"email"`
 	CreatedAt      time.Time `gorm:"column:created_at" json:"createdAt"`
 	PhoneNumber    uint64    `gorm:"column:phone_number" json:"phoneNumber"`
 	PanCard        string    `gorm:"column:pan_card" json:"panCard"`
 	Password       string    `gorm:"column:password" json:"password"`
 	Token          string    `gorm:"column:token" json:"token"`
-	CreationTime   time.Time `gorm:"column:created_at" json:"created_at"`
```

Comment:
Why you removed created_at?

Reply:
It was duplicate field CreatedAt  and CreationTime both were storing same thing.